ros2node/ros2node/verb/info.py

```diff
@@ -80,5 +86,9 @@ def main(self, *, args):
                     node=node, remote_node_name=args.node_name, include_hidden=args.include_hidden)
                 print('  Action Clients:')
                 print_names_and_types(actions_clients)
+                print('  Enclaves:')
+                print_enclaves([n.enclave for n in node_names_with_enclaves
+                    if n.name.full_name == args.node_name])
```

Comment:
i am okay with either way, but we could consider this is one of the option for `ros2 node info`.

Reply:
I wasn't sure which is the best option. My rationale for leaving it in the default output is:

1. Most people are running unsecured systems, and in this case it does not add a lot to the information output
2. If someone is running a secured system, this can be important information, and users might want to frequently see this when they are examining a system.

Reply:
Is there any interest in this, or should I withdraw the PR? Internally, we've found that this information would be very useful for our development, and I'd like to help the community if possible.

Reply:
I think this is helpful. can you open this PR as `Non-Draft` for review?

> Most people are running unsecured systems, and in this case it does not add a lot to the information output

IMO, this is why i came up with option such as `--include-enclaves`.